tests/e2e/configurer/chain/commands.go

```diff
@@ -821,7 +821,7 @@ func (n *NodeConfig) ParamChangeProposal(subspace, key string, value []byte, cha
 			return false
 		}
 		return status == proposalStatusPassed
-	}, time.Minute, 10*time.Millisecond)
+	}, time.Minute*2, 10*time.Millisecond)
```

Comment:
This is done due to a werid fluke in the test suite where after we upgrade, the blocktime is set slightly in the past. So we need to wait for some time to pass before the proposal actually becomes active. I do not see this issue on the state exported testnet but will continue to keep my eye open for it